client/src/utils/stl.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Module to determine if file upload is a valid stl file. Not using ES6 class since server isn't set up to compile
```

Comment:
This comment (about ES6 classes) can be removed and ES6 classes are supported in Node.
The majority of ES6 features are supported by Node with the exception of modules being under experimental support.

Reply:
In this case it wasn't compiling but I see your point on the comment since isn't explicitly correct. The intent was to explain why I opted to go for an object export rather than using a class. In hindsight the comment was unnecessary and could have been noted in a comment.